qa/standalone/scrub/osd-unexpected-clone.sh

```diff
@@ -66,7 +66,10 @@ function TEST_recover_unexpected() {
     objectstore_tool $dir $osd "$JSON" set-bytes data
     objectstore_tool $dir $osd "$JSON" set-attr _ _
 
-    sleep 5
+    wait_for_osd up 0 || return 1
+    wait_for_osd up 1 || return 1
+    wait_for_osd up 2 || return 1
+    wait_for_clean || return 1
```

Comment:
Strange objectstore_tool does a wait_for_clean, but maybe it should do the wait_for_osd up on the osd it just brought back, but that might change the timing of other tests.  I'll try it.